docker/docker-entrypoint.sh

```diff
@@ -16,6 +16,13 @@ if [[ "$1" == "dashd" ]]; then
   else
     echo "Tor control credentials not provided"
   fi
+  # If an masternode private key is provided (length of $CORE_MASTERNODE_OPERATOR_PRIVATE_KEY is nonzero) then dashd should be run as a masternode. Then we add masternodeblsprivkey to config.
+  if [[ -z $CORE_MASTERNODE_OPERATOR_PRIVATE_KEY ]]; then
+    echo "No CORE_MASTERNODE_OPERATOR_PRIVATE_KEY provided, this node is a regular wallet node."
+  else
+    echo "masternodeblsprivkey=$CORE_MASTERNODE_OPERATOR_PRIVATE_KEY" >> "$HOME/.dashcore/dash.conf"
```

Comment:
Not sure this work since we are mounting configs usually. Why not just update the config initially instead of passing the variable?

Reply:
(locally) it works just fine with dashmate.  If not using command-line args is a problem, then using commandline args is also possible. I don't know if the method matters that much, I thought the main goal was letting dashd know about the masternode privatekey iff it exists. Which can be done in many ways, I guess. 
I'll push a dashd arguments version.